CHANGELOG.md

```diff
@@ -116,6 +53,34 @@ As of v3.0.0 this project adheres to [Semantic Versioning](http://semver.org/).
 - `\grecommentary` is now less restrictive about where it occurs and need not occur directly before the score anymore.
 
 ### Added
```

Comment:
Elisions are first mentioned at the end of this section; if we want to mention the bugs fixed with elision, it should be done there, but I don't think it's particularly necessary.